content/docs/tasks/traffic-management/ingress.md

```diff
@@ -31,6 +31,12 @@ This task describes how to configure Istio to expose a service outside of the se
     $ kubectl apply -f samples/httpbin/httpbin.yaml
     ```
 
+    Without _Istio-Initializer_:
```

Comment:
Actually we have replace `Initializer` with `webhook`

Reply:
This is already merged at https://github.com/istio/istio.github.io/pull/1435